File: C++/leibniz.cpp
// ./leibniz
#include <iostream>

using namespace std;

int main(){
	double s = 0;
	int d = 1;
	for(int i = 0;i<100000;i++){
		if(i%2==0)
			s += 4./d;
		else
			s -= 4./d;
		d+=2;
	}
	cout<<s<<endl;
}

File: JavaScript/leibniz.js
// node leibniz.js
var k = 1;
var s = 0;

for(var i = 0;i<100000;i++){
	if (i%2 == 0){
		s += 4/k
	}else
		s -= 4/k
	k += 2
}

console.log(s)